### PR TITLE
Enable doc comments generation

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: src/Components/WebAssembly/Build/src/Microsoft.AspNetCore.Components.WebAssembly.Build.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <IsShippingPackage>true</IsShippingPackage>
     <HasReferenceAssembly>false</HasReferenceAssembly>
     <GenerateDependencyFile>false</GenerateDependencyFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (danroth27) [inline src/Components/WebAssembly/Build/src/Microsoft.AspNetCore.Components.WebAssembly.Build.csproj:11]: Are there any public facing APIs in this package? Maybe this one isn't needed.

File: src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj [hunk 2]
```diff
@@ -15,6 +15,7 @@
     <!-- This is so that we add the FrameworkReference to Microsoft.AspNetCore.App -->
     <UseLatestAspNetCoreReference>true</UseLatestAspNetCoreReference>
     <MicrosoftAspNetCoreAppVersion>3.1.0</MicrosoftAspNetCoreAppVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (danroth27) [inline src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj:18]: I don't think the dev server has any public surface area that we need to document.

File: src/Components/WebAssembly/Build/src/Microsoft.AspNetCore.Components.WebAssembly.Build.csproj (hunk 1 above)
Comment (mkArtakMSFT) [reply src/Components/WebAssembly/Build/src/Microsoft.AspNetCore.Components.WebAssembly.Build.csproj:11]: @pranavkm please let me know whether I need to revert this particular file or not.

**Review by wtgodbe — APPROVED** [review]

**Review by pranavkm — APPROVED** [review]

**Comment by mkArtakMSFT** [comment]
Thanks @pranavkm !

**Pull request merged**